c++/include/orc/OrcFile.hh

```diff
@@ -103,12 +103,18 @@ namespace orc {
   };
 
   /**
-   * Create a stream to a local file.
+   * Create a stream to a local file or HDFS file if path begins with "hdfs://"
```

Comment:
why do you want readLocalFile() to handle hdfs files? You already have readHdfsFile() which does exactly that.

Reply:
Yes, I guess this is a little confusing. Currently I am calling readHdfsFile from readLocalFile if it starts with "hdfs://". To fix this I will add a new function readFile which will call either readHdfsFile or readLocalFile.